### DISCUSSION TITLE
Can we promote powershell to become the default shell for Microsoft Linux distributions?

### DISCUSSION THREAD

**Discussion by kasini3000** [PowerShell/PowerShell#16648] [Show and tell]
hi fans:

To make powershell great again.
even if it is unsuccessful, let Microsoft release the integrated powershell.
In fact, it is sufficient to integrate the Microsoft software source. packages.microsoft.com
Is it already like this? I don't know. There is no news between the Microsoft distribution and powershell.

----

Recently I saw a piece of news:
Microsoft's CBL-Mariner Linux distribution has been used in Azure, Windows Subsystem for Linux, SONiC and other Microsoft teams/departments.

Microsoft has been releasing monthly updates for this RPM-based Linux distribution, and today marks their November 2021 release. With this new update of CBL-Mariner, there are various package upgrades, including DNF 4.10 and libdnf 0.65 for package management, OSTree 2020.1.4, RPM-OSTree 2020.4 and so on. CBL-Mariner has now started to use the latest Linux 5.10 LTS maintenance version.

In addition to regular software package upgrades for prominent software components, there are also new versions and patches that stem from various security issues. The download and complete list of changes to Microsoft's CBL-Mariner Linux distribution can be found on GitHub.

**Comment by szymonos** [comment]
First it needs to get rid of annoying logo at startup.
BTW. PowerShell is my default shell on Linux, but I but that thing is driving me crazy.

**Reply by kasini3000** [reply]
hi beginner:
-NoLogo | -nol
    Hides the copyright banner at startup of interactive sessions.

**Reply by szymonos** [reply]
Lol. I know that switch. But it's not trivial on Linux because you cannot specify arguments when setting pwsh as default with chsh. I have NoLogo set up everywhere it is possible, in VSCode, Windows Terminal for every WSL distribution, Gnome-Terminal, but you cannot do it e.g. in Guake.
There are numerous issues on GitHub with people asking for the feature to switch logo off, and it may happen with PS 7.3 so fingers crossed.

**Comment by rhubarb-geek-nz** [comment]
Hi,

PowerShell should not be a default part of CBL-Mariner as it is supposed to be a very lightweight distribution for running on edge devices and in the cloud for running containers..

Powershell is however already available for x86_64

https://packages.microsoft.com/cbl-mariner/2.0/prod/Microsoft/x86_64/powershell-7.2.6-1.cm.x86_64.rpm

The PowerShell team should release ARM64 for Linux just like x86_64. I have asked but nothing doing.

https://github.com/PowerShell/PowerShell/issues/16777

If you want to get PowerShell popular on Linux then get it available as a package for the Raspberry Pi and in schools.

Regards,

R
